### ISSUE TITLE
unable to download files listed

### ISSUE THREAD

**Issue by csando** [seguler/static-website-blob-browser#6]
Hi
Not sure if this project still has any support, but I'm having a small issue.

The site works fine and enumerates all the files in the container, but clicking the file to download results in a 404 error.

Any tips on how to rectify this? Thanks for the work here BTW this will be useful for me.

**Comment by SoftbayX** [comment]
Hi @csando @seguler 

Did you manage to resolve the download issue? i am having the same issue whereby I can list the item in the blob storage but am not able to download them when they are clicked, instead, it opens another page to view the blob item.

please HELP. I will be very grateful to receive your assistance

**Comment by csando** [comment]
> Hi @csando @seguler
> 
> Did you manage to resolve the download issue? i am having the same issue whereby I can list the item in the blob storage but am not able to download them when they are clicked, instead, it opens another page to view the blob item.
> 
> please HELP. I will be very grateful to receive your assistance

Hey!

Yes, I actually just used this fork which worked perfectly:
https://github.com/msdz/static-website-blob-browser

Let me know how that goes!

The issue I found was that the download URL links that are generated are not correct in this version. The fork I linked to solves that problem!

**Comment by SoftbayX** [comment]
Thank you very much❤️

I have already used this project. It works perfectly, but the project
doesn't download. Only view. Please assist.

Kind Regards

Divin

**Comment by csando** [comment]
> Thank you very much❤️ I have already used this project. It works perfectly, but the project doesn't download. Only view. Please assist. Kind Regards Divin
> […](#)
> On Fri, 24 Sep 2021, 01:06 csando, ***@***.***> wrote: Hi @csando <https://github.com/csando> @seguler <https://github.com/seguler> Did you manage to resolve the download issue? i am having the same issue whereby I can list the item in the blob storage but am not able to download them when they are clicked, instead, it opens another page to view the blob item. please HELP. I will be very grateful to receive your assistance Hey! Yes, I actually just used this fork which worked perfectly: https://github.com/msdz/static-website-blob-browser Let me know how that goes! — You are receiving this because you commented. .

I'm not sure what you mean by it doesn't download. You can't download the code for it?

**Comment by SoftbayX** [comment]
No.  What i mean is that the webapp isn't downloading the blob items. It
displays them but doesn't allow for downloading of them

**Comment by csando** [comment]
@SoftbayX 
I think that is more a browser setting on your side. If it displays them then it can download them.

What if you right click the file and select download?

**Comment by Peiprjs** [comment]
> 
> 
> > Hi @csando @seguler
> > Did you manage to resolve the download issue? i am having the same issue whereby I can list the item in the blob storage but am not able to download them when they are clicked, instead, it opens another page to view the blob item.
> > please HELP. I will be very grateful to receive your assistance
> 
> Hey!
> 
> Yes, I actually just used this fork which worked perfectly: https://github.com/msdz/static-website-blob-browser
> 
> Let me know how that goes!
> 
> The issue I found was that the download URL links that are generated are not correct in this version. The fork I linked to solves that problem!

So I tried but now it's stuck at loading
